nilmtk/dataset_converters/ampds/convert_ampds.py

```diff
@@ -46,18 +46,22 @@ def convert_ampds(input_path, output_filename, format='HDF'):
         Defaults to HDF5
     Example usage:
     --------------
-    convert('/AMPds/electricity', 'store.h5')    
+    convert_ampds('dataverse_files/', 'AMPds.h5')
 
     """
     check_directory_exists(input_path)
     files = [f for f in listdir(input_path) if isfile(join(input_path, f)) and
-             '.csv' in f and '.swp' not in f]
+             '.csv' in f and '.swp' not in f and 'Electricity' in f and 'Billing' not in f and 'Monthly' not in f
+             and  'Electricity_I.csv' not in f   and  'Electricity_Q.csv' not in f   and  'Electricity_P.csv' not in f
```

Comment:
`Electricity_P.csv` contains the active power for all channels already. Why ignoring it?

Reply:
That had been part of the converter already. I cannot explain why they decided to ignore that file. What I can tell for sure is that the converter works.

Reply:
I wanted to check again today but just noticed: there are no more CSV files in https://dataverse.harvard.edu/dataset.xhtml?persistentId=doi:10.7910/DVN/FIE0S4

Reply:
It should work fine when you download the CSV files from Harvard instead of TAB. The reason I pushed this update was mainly due to updates in the metadata, where I corrected labels. Without those corrections to the metadata, AMPds2 cannot be used with NILMTK's API.